docker/efficiency/efficiency.py

```diff
@@ -0,0 +1,113 @@
+from . import builds
+from . import containers
+from . import images
+
+_client = None
```

Comment:
I think this is a bad idea. Globals make libraries much more difficult to use.

I would pass in a client instead of registering a global. It's going to make this API much more re-usable, easier to test, and easier to reason about.